src/studio/screens/mainmenu.c

```diff
@@ -218,7 +242,8 @@ static const MenuItem OptionMenu[] =
     {"VSYNC",           NULL,   &VSyncOption, "VSYNC needs restart!"},
     {"FULLSCREEN",      NULL,   &FullscreenOption},
     {"VOLUME",          NULL,   &VolumeOption},
-    {"SETUP GAMEPAD",   showGamepadMenu},
+    {"SETUP GAMEPAD",       showGamepadMenu},
+    {"CODE EDITOR OPTIONS", showCodeEditorMenu},
```

Comment:
The same here, pls wrap the option declaration with `#if defined(BUILD_EDITORS)`